contrib/msggen/msggen/schema.json

```diff
@@ -12587,6 +12587,34 @@
                   "description": [
                     "A full description of this command (including whether it's deprecated)."
                   ]
+                },
+                "clnrest": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "required": [],
```

Comment:
if you add this are any of the fields required? maybe `path` and `content_type`?

Reply:
Oh yea, this is the response for the help command. So if all of the fields have default values, then these would all be required, yea?

Reply:
I set all these fields to required in [61c471a](https://github.com/ElementsProject/lightning/pull/7507/commits/61c471a342207e5de611dd552e1c3d8ee3b43bf0)

That makes sense right?